server/graphql/v2/mutation/ConnectedAccountMutations.ts

```diff
@@ -37,20 +38,29 @@ const connectedAccountMutations = {
         throw new Unauthorized("You don't have permission to edit this collective");
       }
 
-      if (args.connectedAccount.service === Service.TRANSFERWISE) {
+      if ([Service.TRANSFERWISE, Service.PAYPAL].includes(args.connectedAccount.service)) {
         if (!args.connectedAccount.token) {
-          throw new ValidationFailed('A token is required for TransferWise accounts');
+          throw new ValidationFailed('A token is required');
         }
         const sameTokenCount = await models.ConnectedAccount.count({
-          where: { service: Service.TRANSFERWISE, token: args.connectedAccount.token },
+          where: { token: args.connectedAccount.token },
```

Comment:
Not fan of this one, having the same token shouldn't conflict if we're not using the same service (even if the risk of having a collision is pretty low). Can we do something like:

```suggestion
          where: { token: args.connectedAccount.token, service: args.connectedAccount.service },
```